examples/js/loaders/GLTFLoader.js

```diff
@@ -1910,7 +1910,7 @@ THREE.GLTFLoader = ( function () {
 				if ( bufferView !== null ) {
 
 					// Avoid modifying the original ArrayBuffer, if the bufferView wasn't initialized with zeroes.
-					bufferAttribute.setArray( bufferAttribute.array.slice() );
```

Comment:
I'm assuming we'd encourage users to create a new BufferAttribute rather than overwriting `.array`, so let's do the same here.

Reply:
Make sense!